spec/features/hipstergram_features_spec.rb

```diff
@@ -0,0 +1,35 @@
+require "rails_helper"
+
+RSpec.feature "hipstergrams" do
+
+  before { sign_in(sign_up) }
+  
+  context "When no hipstergrams have been added" do
+
+    scenario "It should display a prompt to upload an hipstergram" do
+      visit "/hipstergrams"
+      expect(page).to have_content "You got here before it was cool; no hipstergrams have been added"
+      expect(page).to have_link "Upload new hipstergram"
+    end
+  end
+
+
+  context "When an hipstergram has been added" do
+    
+    scenario "An hipster can upload an hipstergram" do
+        visit '/posts'
+        click_link 'Pick a photogram'
+        attach_file :post_image, './spec/images/test.jpg'
+        fill_in 'Description', with: 'Cold brew triple distilled mason water coffee'
```

Comment:
Metrics/LineLength: Line is too long. [84/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)